CFC_WebApp/api/cfc_webapp.py

```diff
@@ -382,7 +401,10 @@ def getUUID(request):
     retUUID = user_uuid
     logging.debug("skipAuth = %s, returning fake UUID %s" % (skipAuth, user_uuid))
   else:
-    userToken = request.json['user']
+    if inHeader:
+      userToken = request.headers.get('Authorization').split()[1]
```

Comment:
Woops. It is 'User'. So I will need to change this, and the phone side code to reflect the change from 'user' to 'User'.